scripts/_retag.sh

```diff
@@ -1,19 +1,23 @@
 #!/bin/bash
 
-set -euxo pipefail
+set -euo pipefail
 
 # This is a script to re-tag a Docker image without pulling the pushing the image but rather directly in the registry.
 
 # Make sure our docker registry string ens with "/" so it works with the registry string that we use in other places
 if [[ "$DOCKER_REGISTRY" != */ ]]; then
-    DOCKER_REGISTRY="${DOCKER_REGISTRY}/" 
+    DOCKER_REGISTRY="${DOCKER_REGISTRY}/"
 fi
 
-TOKEN=$(aws ecr get-authorization-token --region eu-west-1 --output text --query 'authorizationData[].authorizationToken')
+if [[ -z "$TOKEN" ]]; then
```

Comment:
I think there would be an error if $TOKEN is not defined

Reply:
ah yes. forgot that.